js/MarkupSubmissionsBatchConversion.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * @file js/MarkupSubmissionsBatchConversion.js
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2000-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @package plugins.generic.markup
+ * @class MarkupSubmissionsBatchConversion
+ *
+ * @brief Batch conversion client side handler
+ */
+(function($) {
+
+	/** @type {Object} */
+	$.pkp.plugins.markup =
+			$.pkp.plugins.markup ||
+			{ js: { } };
+
+	/**
+	 * @constructor
+	 */
+	$.pkp.plugins.markup.js.MarkupSubmissionsBatchConversion = function() {
+		// submission selection click handler
+		$('div#markupBatchConversionGridContainer').on('click', 'input:checkbox[id*="select-"]', function() {
+		    var $el = $(this);

```

Comment:
Watch out for space-based indents -- grep/replace throughout, and make sure your editor is set to use tabs.